### ISSUE TITLE
InsetDrawable cannot be cast to TransitionDrawable

### ISSUE THREAD

**Issue by mrfleap** [flipkart-incubator/springy-heads#14]
I've implemented the run_in_service library into my project. At the moment when I click on the springy head to maximize the arrangement, it crashes and spits out an error. The main line of the error is `java.lang.ClassCastException: android.graphics.drawable.InsetDrawable cannot be cast to android.graphics.drawable.TransitionDrawable`

Error: [Here](https://gist.github.com/mrfleap/2afe7c57f8a9b2341546bb410013db63)

Chat head service class: [Here](https://gist.github.com/mrfleap/28171e49af3fa6a0ecf8b5bc102415e8)

**Comment by thekirankumar** [comment]
Looks like a resource conflict. By any chance do you have a drawable named overlay_transition in your project ?

**Comment by mrfleap** [comment]
The only drawables I have added are the circular_ring.xml and circular_view.xml.

**Comment by thekirankumar** [comment]
Since you have cloned the repo, can you rename overlay_transition.xml using studio's refactor menu  and try running ?

**Comment by mrfleap** [comment]
I believe the way I'm implementing the library is incomplete. Would I download the source code to my project and add it to the app's gradle file to use the library? I found the `compile 'com.flipkart.springyheads:library:0.9.6'` is only the master branch.

**Comment by thekirankumar** [comment]
Yes, right now the only way you can use run_in_service branch is to clone the source.

**Comment by thekirankumar** [comment]
You are welcome
